### ISSUE TITLE
Suggestion: Days

### ISSUE THREAD

**Issue by JomiliDesign** [CallMeFoxie/CanIJoinNow#2]
Hi CallMeFoxie,

are you interested in extending your mod - _showing the current days?_

Would be awesome if you could make some options in the config for such a thing, because maybe not all users are interested in this capability.

Example options:
```
B:timeActive=true
B:timeFirst=true
B:daysActive=true
S:daysPrefix="Day "
S:daysSuffix=""

Would lead to: 13:30 Day 5
```

```
B:timeActive=true
B:timeFirst=false
B:daysActive=true
S:daysPrefix=""
S:daysSuffix=". day "

Would lead to: 5. day 13:30
```

I'm asking because I'm using the mod "[Bloodmoon](https://minecraft.curseforge.com/projects/bloodmoon)" and the mod has some configs, when the Bloodmoon starts. One config is that the Bloodmoon starts every n-th night which I'm using. For me and my friends it would be awesome if they could see before joining the server if they are joining into a Bloodmoon.
(The mathematic division of 'current day' / 'n-th' they need to do on their own ^^)

I'm using GitHub for this suggestion, because you've written that you most likely will not notice Curse.

Kind regards

**Comment by CallMeFoxie** [comment]
Hi
I will look into it. Is 1.12.(1) MC version okay?

I would however not do it as B/S mix, rather than similar to linux' `date` syntax :)

**Comment by JomiliDesign** [comment]
Hi CallMeFoxie,

what a fast reply. :-)
Yes, I'm talking about 1.12(.1). Currently I'm on "forge-1.12.1-14.22.0.2452-universal" with "CanIJoinNow-1.12.1-1.00".

**Comment by CallMeFoxie** [comment]
This and the previous thing are released as version 1.1 :) please give it a go!
